index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome/css/font-awesome.min.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="flex-nav">
+        <h1 class="logo">My Logo</h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
+        <p>
+          I&rsquo;m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don&rsquo;t hesitate to contact
+          me.
+        </p>
+        <p class="connect">LET’S CONNECT</p>
```

Comment:
Your design
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/eb14f6a4-f749-4420-9e05-6d0e4ef672b7)

Figma
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/caf090fa-e3f6-4d14-ae5d-5603ee1c2291)

Kindly make the text `LET'S CONNECT` a link `a` tag and let's follow the design guidelines (e.g. font-wieght and colors) 💪

Apart from being semantically correct, the tag will let you connect to other sections of the website, using `href`. 👍